### PR TITLE
Close CPPL-153

### PR DESCRIPTION
**Pull request by sorbeee**

### CODE REVIEW

File: src/Client/Models/List/ParserAndFormatter/ListEntity.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#pragma once
```
Comment (Denys-Bykov-0) [inline src/Client/Models/List/ParserAndFormatter/ListEntity.h:1]: Why this file still in Client?

File: src/Client/Models/List/ParserAndFormatter/ListJSONFormatter.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#include "ListJSONFormatter.h"
```
Comment (Denys-Bykov-0) [inline src/Client/Models/List/ParserAndFormatter/ListJSONFormatter.cpp:1]: Why this file still in Client?

File: src/Client/Models/List/ParserAndFormatter/ListJSONFormatter.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#include <QJsonObject>
```
Comment (Denys-Bykov-0) [inline src/Client/Models/List/ParserAndFormatter/ListJSONFormatter.h:1]: Why this file still in Client?

File: src/Client/Models/List/ParserAndFormatter/ListJSONParser.cpp [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#include "ListJSONParser.h"
```
Comment (Denys-Bykov-0) [inline src/Client/Models/List/ParserAndFormatter/ListJSONParser.cpp:1]: Why this file still in Client?

File: src/Client/Models/List/ParserAndFormatter/ListJSONParser.h [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#include <QJsonObject>
```
Comment (Denys-Bykov-0) [inline src/Client/Models/List/ParserAndFormatter/ListJSONParser.h:1]: Why this file still in Client?

File: src/Net/Entities/ListState.h [hunk 6]
```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "Types.h"
+
+template<typename  T>
```
Comment (Denys-Bykov-0) [inline src/Net/Entities/ListState.h:5]: Remove redundunt whitespace

File: src/Net/Tools/Formatters/List/ListIdJSONFormatter.h [hunk 7]
```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <QJsonObject>
+
+#include "../Net/Entities/List.h"
+#include "../Net/Parsing.h"
+
+template<class Entity> // rename
```
Comment (Denys-Bykov-0) [inline src/Net/Tools/Formatters/List/ListIdJSONFormatter.h:7]: Remove // rename

File: src/Net/Tools/Formatters/List/ListJSONFormatter.h [hunk 8]
```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <QJsonObject>
+
+#include "../Net/Entities/List.h"
+#include "../Net/Parsing.h"
+#include "ListStateJSONFormatter.h"
+
+template<class Entity> // rename
```
Comment (Denys-Bykov-0) [inline src/Net/Tools/Formatters/List/ListJSONFormatter.h:8]: Remove // rename

File: src/Net/Tools/Formatters/List/ListsJSONFormatter.h [hunk 9]
```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <QJsonObject>
+
+#include "../Net/Entities/List.h"
+#include "../Net/Parsing.h"
+#include "ListJSONFormatter.h"
+
+template<class Entity> // rename
```
Comment (Denys-Bykov-0) [inline src/Net/Tools/Formatters/List/ListsJSONFormatter.h:8]: Remove // rename

File: src/Server/HandlerFactory.cpp [hunk 10]
```diff
@@ -26,7 +26,7 @@ Poco::Net::HTTPRequestHandler* HandlerFactory::createRequestHandler(const Poco::
 	}
 	catch(const std::exception& exception){
 		std::cout << exception.what() << std::endl;
-	}
+    }
```
Comment (Denys-Bykov-0) [inline src/Server/HandlerFactory.cpp:29]: Why are you trying to break indentation?

**Review by Denys-Bykov-0 — APPROVED** [review]

**Review by VasylDobrovolskyi — APPROVED** [review]

**Pull request merged**